src/components/Header/Header.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import './Header.css';
+
+
+const Header = () => (
```

Comment:
I'm glad we can use the single-expression implied-return arrow function here! It removes many unnecessary lines!